src/chunkserver/config_info.h

```diff
@@ -86,6 +86,8 @@ struct CopysetNodeOptions {
     uint32_t port;
     // chunk文件的大小
     uint32_t maxChunkSize;
+    // WAL segment file size
+    uint32_t maxWalSegmentSize;
```

Comment:
should be maxWalChunkSize chunksize+metaPageSize ?

Reply:
YES